Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM roland/on-core
```

Comment:
Oh, I see you have your own stuff in Docker hub. Perhaps we should make a RackHD user there or something?